robottelo/cli/repository.py

```diff
@@ -13,8 +13,10 @@
 
     create                        Create a repository
     delete                        Destroy a repository
+    export                        Export a repository
     info                          Show a repository
     list                          List of repositories
+    remove-content
```

Comment:
Can you add a description here? `Remove content from the repository` just to follow the same pattern as the others.